src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -89,21 +87,21 @@ def __init__(self, rq: dict):
     def job_id(self):
         if PARAM["JOB_ID"] in self.rq_data:
             return self.rq_data[PARAM["JOB_ID"]]
-        raise JobRequestException(JOB_NOT_PROVIDED_ERR)
+        raise JobRequestException(ONE_INPUT_TYPE_ONLY_ERR)
```

Comment:
The frontend doesn't care about these strings, and since it prints the incoming request (which will show the problem with the input), we may as well reduce complexity here and use the same string for all these errors.

Reply:
Oh ...